src/NativeScript/ObjC/Constructor/ObjCConstructorBase.mm

```diff
@@ -13,10 +13,12 @@
 #include "ObjCMethodCall.h"
 #include "ObjCConstructorCall.h"
 #include "ObjCConstructorNative.h"
+#include "../ObjCPrototype.h"
```

Comment:
You can remove the here (and bellow) `../` to be consistent with other header includes. Headers are searched recursively in all source directories.